app/views/publishers/vacancies/job_applications/show.html.slim

```diff
@@ -25,6 +25,44 @@
         = govuk_link_to t("buttons.reject"), organisation_job_job_application_reject_path(vacancy.id, job_application.id), button: true, class: "govuk-button--warning govuk-!-margin-right-3"
       = govuk_link_to t("buttons.download_application"), "#", button: true, class: "govuk-button--secondary"
 
+    h2.govuk-heading-m = "Application sections"
+
+    ul.govuk-list
+      li.app-page-navigation__item = govuk_link_to t(".personal_details.heading"), "#personal_details_summary", class: "govuk-link govuk-body-s govuk-!-margin-bottom-0"
```

Comment:
looks like the govuk-!-margin-bottom-0 style is repeated on every li, i think would be better in SASS file in approprate place